tests/pytests/pkg/integration/test_salt_call.py

```diff
@@ -3,6 +3,10 @@
 import pytest
 from pytestskipmarkers.utils import platform
 
+pytestmark = [
+    pytest.mark.skip_unless_on_linux,
+]
+
```

Comment:
Some of these tests should run on Windows and MacOS. The local ones should still run. Maybe skip just the ones that require a master.

Reply:
You are forgetting what I said in Standp, about the package tests not working due to Caleb getting cut, and in order to move my PR along, skip them and have them fixed later, since unrelated to my PR fixes and tests, and I need to get my PR into the next minor release,  See https://github.com/saltstack/salt/issues/66672 which is Tech Debt to fix these broken tests.  Already tried fixing them for a couple of days and failed, hence they need more time.  Those tests also need to be re-architect'd, (imho), to test the actual package installed and not the pytest factory implementations.